### ISSUE TITLE
`TrueColor` is no longer working on `WindowsDriver`

### ISSUE THREAD

**Issue by tig** [gui-cs/Terminal.Gui#3164]
I probably broke this. Sigh.

**Comment by BDisp** [comment]
I already saw that. My workaround was this:

```cs
		_isWindowsTerminal = Environment.GetEnvironmentVariable ("WT_SESSION") != null ||
			Environment.GetEnvironmentVariable ("VSAPPIDNAME") != null;

```

**Comment by dodexahedron** [comment]
Can't just check OS platform instead?

**Comment by tig** [comment]
> Can't just check OS platform instead?
> 
> ```cs
> OperatingSystem.IsWindows ( )
> ```

Nope. Windows Terminal != conhost etc... We've spent a LOT of effort on this and still don't have it right (and the sands below us are moving)...

- #3164 
- #2962 
- #2928 

To name a few...

**Comment by tig** [comment]
> > Can't just check OS platform instead?
> > ```cs
> > OperatingSystem.IsWindows ( )
> > ```
> 
> Nope. Windows Terminal != conhost etc... We've spent a LOT of effort on this and still don't have it right (and the sands below us are moving)...
> 
> * [Fixes #2961. v2 - WindowsDriver isn't render well on conhost terminal size. #2962](https://github.com/gui-cs/Terminal.Gui/pull/2962)
> * [In the v2 version, using Chinese will cause UI confusion. #2928](https://github.com/gui-cs/Terminal.Gui/issues/2928)
> 
> To name a few...

I have ideas of how to being far more graceful about this in 

- #2610
